test/integration/features/addon-eap-cd.feature.bkp

```diff
@@ -0,0 +1,34 @@
+@addon-eap-cd @addon
+Feature: eap-cd add-on
+  Eap-cd add-on imports eap-cd imagestreams and templates, which are then available in OpenShift to the user.
+
+  Scenario: User enables the eap-cd add-on
+    When executing "minishift addons enable eap-cd" succeeds
+    Then stdout should contain "Add-on 'eap-cd' enabled"
+
+  Scenario: User starts Minishift
+    Given Minishift has state "Does Not Exist"
+    And image caching is disabled
+    When executing "minishift start --memory 4GB" succeeds
+    Then Minishift should have state "Running"
+    And stdout should contain
+      """
+      EAP CD imagestreams and templates installed
+      """
+
+  Scenario Outline: User deploys, checks out and deletes several templates from EAP CD imagestreams and templates
+    Given Minishift has state "Running"
+    When executing "oc new-app <template-name>" succeeds
+    And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
+    And service "<service-name>" rollout successfully within "1200" seconds
+    Then with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" contains "<expected-hello>"
+    And with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
+    And executing "oc delete all --all" succeeds
+
+    Examples: Required information to test the templates
+      | template-name           | service-name   | http-endpoint | expected-hello                        |
+      | eap-cd-basic-s2i        | eap-app        | /             | Welcome to JBoss!                     |
+
+  Scenario: User deletes Minishift
+    When executing "minishift delete --force" succeeds
+    Then Minishift should have state "Does Not Exist"
```

Comment:
@budhrg I guess this is a mistake to have `test/integration/features/addon-eap-cd.feature.bkp` in the PR.